nifi-nar-bundles/nifi-aws-bundle/nifi-aws-abstract-processors/pom.xml

```diff
@@ -88,6 +98,11 @@
             <groupId>org.apache.nifi</groupId>
             <artifactId>nifi-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-security-utils</artifactId>
```

Comment:
The `nifi-security-utils` module includes a number of unnecessary items. Instead of using `KeyStoreUtils`, the builders in `nifi-security-ssl` should be used, avoiding this dependency.